package.json

```diff
@@ -10,42 +10,52 @@
     "src"
   ],
   "scripts": {
-    "build": "npm run test && npm run clean && npm run lint-js && npm run build-es5 && npm run build-webpack",
-    "build-doc": "jsdoc  --package package.json -d doc ./src/**/*.js",
-    "build-es5": "babel src --out-dir lib ",
-    "build-webpack": "webpack -p --progress --colors --display-error-details",
+    "babel": "babel src --out-dir lib --ignore '**/test/*'",
+    "build": "yarn run babel && yarn run webpack",
     "clean": "rm -rf dist lib doc",
     "dev": "webpack-dev-server --config webpack.config.dev.js",
-    "lint-js": "eslint 'src/**/*.js' --fix",
-    "test": "mocha --compilers js:babel-core/register --recursive"
+    "docs": "jsdoc  --package package.json -d doc ./src/**/*.js",
+    "lint": "eslint './src/**/*.js'",
+    "prepublish": "yarn run build",
+    "prebuild": "yarn run clean && yarn run lint && yarn run test",
```

Comment:
Lets keep the npm references and npm-package.lock file

Reply:
Why have you changed / remove scripts that haven't been replaced? I might be missing them.